README.md

```diff
@@ -1 +1,15 @@
-# Twitter-clone-Hiba-
\ No newline at end of file
+# Twitter-clone-Hiba-
+
+<h1> Why we do this project? </h1> 
+<p>This project creats to complete one of the requirements of " Web Fundamentals Course " with Gaza Sky Geeks,
```

Comment:
Remove the README.md and push it on a different branch
fix the problems that we talked about!!

Reply:
resolve the conflict too